src/components/cesium_map/api/spatial.js

```diff
@@ -165,13 +165,17 @@ export class PointStreamPrimitiveCollection extends Cesium.PointPrimitiveCollect
           } else {
             locations[location] = 1;
           }
+          if (Array.isArray(doc.z)){
+            doc.z = doc.z[0] // get integer value if multivalue field
+          }
           const p0 = Cesium.Cartesian3.fromDegrees(doc.x, doc.y, (doc.z || DEFAULT_ELEVATION) + locations[location]);
           this.add({
             id: doc.id,
             position: p0,
             color: Cesium.Color.fromCssColorString(colorbind[CV.indexOf(doc[field]) % colorbind.length]),
             pixelSize: 8,
-            disableDepthTestDistance: 1
+            // disable the depth test
+            disableDepthTestDistance: Number.POSITIVE_INFINITY,
```

Comment:
The value of this key is the distance from the camera we want to disable the depth test to. By setting it to infinite value, the depth test is never applied, which allows the entities to appear at the top always.

Reply:
This has the unfortunate side effect of points being visible through terrain. So while it does fix the issue of points getting hidden by terrain incorrectly, it also never hides them which can be a bit confusing.